float32 timestamp
int32 packetsSent
int32 bytesSent
int32 framesEncoded
int32 packetsReceived
int32 bytesReceived
int32 headerBytesReceived
float32 lastPacketReceivedTimestamp
int32 frameWidth
int32 frameHeight
int32 framesPerSecond
int32 framesDecoded
int32 framesReceived
int32 keyFramesDecoded
int32 framesDropped
float32 totalDecodeTime
float32 totalSquaredInterFrameDelay
float32 bandwidthReceived
float32 currentRoundTripTime
int32 firCount
float32 bandwidthSent
int32 framesDecodedPerSecond
